private/buf/cmd/buf/command/alpha/plugin/pluginpush/pluginpush.go

```diff
@@ -315,6 +315,7 @@ func run(
 		bufplugin.PluginRegistryToProtoRegistryConfig(plugin.Registry()),
 		nextRevision,
 		outputLanguages,
+		pluginConfig.LicenseSpdxID,
```

Comment:
nit: Similar to 'HTTP', 'JSON', etc., I think the style should be to have 'SPDX' in all caps. Perhaps we could rename this field to `SPDXLicenseID`?